organization/post-workshop-survey.md

```diff
@@ -0,0 +1,69 @@
+# Post-Workshop Survey
+
+1. Did you attend Building the Peer-to-Peer Internet?
```

Comment:
Isn't this always a yes? A little confusing.

Reply:
It is yes, it is basically an explicit opt-in on the form, if people don't feel like agreeing to it then we don't use the data

Reply:
But this is asking if they attended not consent to this form.